### DISCUSSION TITLE
Anime game version

### DISCUSSION THREAD

**Discussion by BrownieDog** [Anime-Game-Servers/AGS-GC#70] [Q&A]
What anime game version does this fork support? 

As-is my handbook generation is failing because it is searching for a resource file that doesn't exist. Is this because of a version mismatch? I have the latest resource package (pulled today) and extracting the custom resources does not add the missing file (.\resources\ExcelBinOutput\DungeonElementChallengeExcelConfigData.tsj)

**Comment by ghost** [comment]
3.2 iirc
